pipeline/data_store.py

```diff
@@ -540,6 +548,20 @@ def get_image(self, provenance=None, session=None):
 
         return self.image  # could return none if no image was found
 
+    def apped_image_products(self):
```

Comment:
```suggestion
    def append_image_products(self):
```

Reply:
Yes, this was fixed in one of the later PRs.